source/pervasive/option.rs

```diff
@@ -35,6 +35,24 @@ impl<A> Option<A> {
         }
     }
 
+    pub const fn is_some(&self) -> (res: bool)
```

Comment:
We probably want to add this annotation (and similar for `is_none`)
```suggestion
    #[verifier(when_used_as_spec(is_Some))]
    pub const fn is_some(&self) -> (res: bool)
```

Reply:
Agree, that would be great. Seems it doesn't quite work. Adding this, I'm getting the following error. The function is generated by `#[is_variant]` on the struct, maybe that's the reason? 

```
error: cannot find function referred to in when_used_as_spec
  --> nr/src/pervasive/option.rs:39:5
   |
39 |     pub const fn is_some(&self) -> (res: bool)
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
yeah that doesn't surprise me, is_Some is treated as a special function

Reply:
Please add `#[inline(always)]` to both.

Reply:
I've added the `#[inline(always)]` to both.